src/TwigComponent/src/ComponentFactory.php

```diff
@@ -124,6 +133,12 @@ private function mount(object $component, array &$data): void
             return;
         }
 
+        if ($component instanceof AnonymousComponent) {
+            $component->mount($data);
+
+            return;
+        }
```

Comment:
Hmm. It's not mandatory, but I'd love for `AnonymousComponent` to not need a special case. Maybe this is possible:

```php
public function mount(...$props = [])
{

}
```

Tell me if this makes sense: if the final argument of `mount()` is a `...$props` variadic-style argument, then we pass everything. This should also be possible:

```
public function mount(string $name, ...$props = [])
{

}
```

Reply:
I tried but looks like we just need a little tweak here if we don't wan't to move too much code.